[echarts] Add appendToBody, className and order property to tooltip

bhaskar20 opened the pull request:
Please fill in this template.

Documentation: 
appendToBody: https://echarts.apache.org/en/option.html#tooltip.appendToBody

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

Ky6uk commented:
ECharts 5 has its own type definitions. I don't think the version bump to v5 makes sense here. I just removed `@types/echarts` completely from `package.json` after upgrading to ECharts 5 and it works fine.

Also, the style in that package is single quotes.

bhaskar20 commented:
@Ky6uk 
> ECharts 5 has its own type definitions. I don't think the version bump to v5 makes sense here. I just removed @types/echarts completely from package.json after upgrading to ECharts 5 and it works fine.

Makes sense, i have removed the changes related to ECharts 5.

> Also, the style in this package is single quotes.

I see double quotes being enforced in prettier config (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51436)

bhaskar20 commented:
Ready to merge
